pkg/tuned/dialog.js

```diff
@@ -266,6 +272,10 @@
             tuned_service.start()
                 .done(function () {
                     update_button();
+                    /* There are a few cases where tuned can be started as user,
```

Comment:
This is a bit confusing - better say "by an user", as tuned will still run as root.